scripts/serial_link.py

```diff
@@ -334,6 +360,36 @@ def format_log_entry(t0, item):
   delta = int((time.time() - t0)*1000)
   return (delta, timestamp, item)
 
+def format_log_json_entry(t0, item):
+  """
+  Generates a Python object with logging metadata.
+
+  Parameters
+  ----------
+  t0 : float
+    Reference time (seconds)
+  item : object
+    Python object to serialize
+
+  Returns
+  ----------
+  {'delta': delta, 'timestamp': timestamp, 'data': data} : dict
+    delta = msec reference timestamp (int)
+    timestamp = current timestamp (int - UTC epoch)
+    data = Python object to JSON serialize
+  """
+  timestamp = calendar.timegm(time.gmtime())
+  delta = int((time.time() - t0)*1000)
+  data = {
```

Comment:
Ah yeah, sorry: that's actually closer to what I meant (I should have been more specific). The creation of the dict from the SBP attributes is separate concern from the timestamped JSON record that you're emitting. I think `libsbp` will also need some kind of corresponding `to_json_dict`/`from_json_dict` methods for reifying SBP objects.